zerver/migrations/0401_create_role_based_groups_for_internal_realms.py

```diff
@@ -0,0 +1,159 @@
+# Generated by Django 3.2.13 on 2022-06-28 17:36
+
+from django.conf import settings
+from django.db import migrations, transaction
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.utils.timezone import now as timezone_now
+
+
+def create_role_based_system_groups_for_internal_realms(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
```

Comment:
This migration is just basically just a copy of `zerver/migrations/0382_create_role_based_system_groups.py` run for the system bot realm, right? Seems worth it to note that in a comment somewhere in the file